hexrd/material/unitcell.py

```diff
@@ -616,7 +617,7 @@ def remove_duplicate_atoms(self,
         """
 
         if atom_pos is None:
-            atom_pos = self.atom_pos
+            atom_pos = copy.deepcopy(self.atom_pos)
```

Comment:
Since you're not modifying `atom_pos`, I don't think this deep copy is necessary, is it?

Reply:
I saw this earlier as well.  At the end of this function you actually overwrite self.atom_pos anyway, so it shouldn't matter at all

Reply:
Thanks for catching this. I'll fix it.